schemas/json/no_service_plan.erb

```diff
@@ -1,8 +1,14 @@
 {
   "service_offering_id": "<%= @reference %>",
   "id": "DNE",
-  "create_json_schema": {
-    "type": "object",
-    "properties": {}
-  }
+  "create_json_schema": <%= {
+    "schema": {
+      "fields": [{
+        "component": "plain-text",
+        "name": "plain-text-component",
```

Comment:
This name should probably somehow represent the purpose. Something like `empty-service-plan` or something similar.